dependencies.gradle

```diff
@@ -5,8 +5,12 @@ dependencies {
     api('com.github.GTNewHorizons:GTNHLib:0.6.30:dev')
 
     compileOnly('com.github.GTNewHorizons:BuildCraft:7.1.43:api') {transitive=false}
-    compileOnly('com.github.GTNewHorizons:ForgeMultipart:1.6.4:dev') {transitive=false}
-    devOnlyNonPublishable('com.github.GTNewHorizons:NotEnoughItems:2.7.51-GTNH:dev')
+    compileOnly('com.github.GTNewHorizons:ModularUI2:2.2.15-1.7.10:dev')
+//    runtimeOnlyNonPublishable('com.github.GTNewHorizons:ModularUI2:2.2.15-1.7.10:dev')
```

Comment:
Please cleanup dead code.

Reply:
That being said, having it be `devOnlyNonPublishable` instead is equivalent in terms of ability to toggle it through commenting/uncommenting so that might be  preferable  to committing a commented out `runtimeOnlyNonPublishable`.